src/components/SnowFlakes.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import { useResponsiveValue } from '../hooks/useResponsiveValue';
+import { Snowfall, Snowflake } from 'react-snowflakes';
+import '../styles/SnowFlakes.css';
+
+const SnowFlakes = () => {
+    // TODO: rewrite component without library
```

Comment:
You can, but don't have to do this. If the snowflakes look good enough with this library, I don't see why you would need to rewrite this.

Reply:
They don't. That's why it is in TODO

Reply:
Complete all TODOs before merging. Or remove the snowflakes for now, they aren't the most important feature and you can add them later.